packages/carbon-component-mapper/rollup.config.js

```diff
@@ -36,7 +36,10 @@ const globals = {
   'prop-types': 'PropTypes',
   '@data-driven-forms/react-form-renderer': '@data-driven-forms/react-form-renderer',
   'carbon-components-react': 'carbon-components-react',
-  '@carbon/icons-react': '@carbon/icons-react'
+  '@carbon/icons-react': '@carbon/icons-react',
+  'carbon-components-react/lib/components/StructuredList/StructuredList': 'carbon-components-react/lib/components/StructuredList/StructuredList',
```

Comment:
this should be covered by the wildcard. Otherwise, it would not work for imports transformed by the plugin.

Reply:
It was showing a warning during building. “Guessing...”

Reply:
:D ok? I guess it has a better guess when compiling the transformed guess. :shrug: